rpc/rpc-wrappers.go

```diff
@@ -539,6 +560,16 @@ func (rac RegistrationAuthorityClient) DeactivateAuthorization(ctx context.Conte
 	return err
 }
 
+// DeactivateAuthorization deactivates a currently valid registration
```

Comment:
typo: You say `DeactivateAuthorization` in this comment when I think you mean `DeactivateRegistration`